src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -270,6 +270,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help='The minimum TLS version to be permitted on requests to storage. '
                         'The default interpretation is TLS 1.0 for this property')
         c.argument('allow_shared_key_access', allow_shared_key_access_type)
+        c.argument('edge_zone', edge_zone_type, min_api='2020-08-01-preview')
```

Comment:
Does it take any string or have a allowed values' list? `microsoftrrdclab1` and `microsoftlosangeles1` from tests don't seem normal.

Reply:
There is no way to get allowed values now unless you know service design. We will add a document link here when service team is ready.

Reply:
`is_preview` is global while `min_api` is not global. I am confused.

Reply:
as different RP has different api version restriction, so we cannot make it global.

Reply:
I mean `is_preview` should not be global.